homeassistant/components/rflink.py

```diff
@@ -345,6 +345,7 @@ def _availability_callback(self, availability):
 
     async def async_added_to_hass(self):
         """Register update callback."""
+        await super().async_added_to_hass()
```

Comment:
Remove the super call to `async_added_to_hass` in the other entity class in this module. That one looks not needed. It is needed here.

Reply:
Hi Martin,
thanks for your review.

Just to clarify. 
This line (348) is from `RflinkDevice(Entity)` and I believe too that it's needed (it is THE bug)

The other entity class you refer is `SwitchableRflinkDevice(RflinkCommand, RestoreEntity)` and you say it's not needed, right?
But, I need to call the `RestoreEntity.async_added_to_hass()` function to make work all the RestoreEntity code. It's not the way to call it? If I remove de `super().async_added_to_hass()` call will be OK?

I'm very, very sorry for arguing at code level, but I screwed up all my environment (again) and can not run tests to validate.

Reply:
You're correct, we need to call it there too, since the RestoreEntity class' `async_added_to_has` will be a parent of that class in the method resolution order. So ignore this comment.

Reply:
Thanks a lot for the clarification and for your patience.